lib/matplotlib/tests/test_axes.py

```diff
@@ -8861,3 +8861,15 @@ def test_axhvlinespan_interpolation():
     ax.axhline(1, c="C0", alpha=.5)
     ax.axhspan(.8, .9, fc="C1", alpha=.5)
     ax.axhspan(.6, .7, .8, .9, fc="C2", alpha=.5)
+
+
+@check_figures_equal()
+def test_latex(fig_test, fig_ref):
```

Comment:
```suggestion
def test_latex_pie_percent(fig_test, fig_ref):
```

Just a slightly more descriptive name, as we are not testing latex generally (and especially not on the axes as a whole)

Reply:
Yeah, makes more sense.